src/Monitoring/Sdk/README.md

```diff
@@ -22,8 +22,9 @@ e.g.
     -p:GrafanaAccessToken=GRAFANA_ADMIN_API_KEY \
     -p:GrafanaKeyVaultName=dotnet-grafana \
     -p:GrafanaKeyVaultAppId=2bdfceef-194a-4775-99d9-b5575c77bc6b \
+    -p:ParametersFile=parameters.json \
     -p:GrafanaKeyVaultAppSecret=KEY_VAULT_APP_SECRET \
-    -p:GrafanaEnvironment=DEPLOYMENT_ENVIRONMENT
+    -p:GrafanaEnvironments=DEPLOYMENT_ENVIRONMENT
```

Comment:
I believe this should not be changed; it should be `GrafanaEnvironment`.

Reply:
I changed it because on the sdk.targets on the MonitoringImport section it was defined as 'GrafanaEnvironments', is this for something different?

Reply:
This command is Publish, not Import. 

It is required for `Import` but gets defined automatically (not needed at the command line) from the props file [here](https://github.dev/dotnet/arcade-services/blob/4e0045f82d7391315c16ec04de3918222e595fe5/src/Monitoring/Sdk/sdk/Sdk.props#L26-L29).